tools/tsan-suppressions

```diff
@@ -1,6 +1,5 @@
 race:libglib*
 race:libgio*
-race:testing::internal::FunctionMockerBase
```

Comment:
what about this? Leave it in? Remove it? I'm not sure what it's used for but I noticed it also references FunctionMockerBase so I didn't just want to ignore it

Reply:
You can remove it; this is just in the TSan suppression lists, and we're (sadly) not TSan clean, nor can we reasonably become so until TSan can understand GLib's locking primitives.